ledger/ledger-api-common/src/main/scala/com/digitalasset/ledger/api/validation/CommandsValidator.scala

```diff
@@ -79,7 +79,7 @@ final class CommandsValidator(
         )
       deduplicationPeriod <- validateDeduplicationPeriod(
         commands.deduplicationPeriod,
-        maxDeduplicationTime,
+          maxDeduplicationDuration,
```

Comment:
Oh, also here.

Reply:
hmmm, weird indeed, I run the format script right before pushing